app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
```

Comment:
`params[:session][:email]` cái ni mà null thì gọi donwcase ổn em nhỉ =))

```suggestion
    user = User.find_by email: params.dig(:session, :email)&.downcase
```